doc/testing-your-application.rst

```diff
@@ -67,15 +50,61 @@ You can override getters in your test class in the following way::
         }
     }
 
+VarnishTestCase
+------------------
+
+Configuration
+''''''''''''''''''
+
+By default, the ``VarnishTestCase`` starts and stops a Varnish server for you.
+All you have to do is to set your Varnish configuration (VCL) file. 
+
+======================= ========================= ============ ==========================================
+Constant                Getter                    Default      Description
+======================= ========================= ============ ==========================================
+``VARNISH_FILE``        ``getConfigFile()``                    your Varnish configuration (VCL) file
+``VARNISH_BINARY``      ``getBinary()``           ``varnishd`` your Varnish binary
+``VARNISH_PORT``        ``getCachingProxyPort()`` ``6181``     port Varnish listens on
+``VARNISH_MGMT_PORT``   ``getVarnishMgmtPort()``  ``6182``     Varnish management port
+``WEB_SERVER_HOSTNAME`` ``getHostName()``                      hostname your application can be reached at
+======================= ========================= ============ ==========================================
+
 Enable Assertions
-~~~~~~~~~~~~~~~~~
+''''''''''''''''''
 
 For the `assertHit` and `assertMiss` assertions to work, you should add a
 :ref:`custom X-Debug header <varnish_debugging>` to responses served
 by your Varnish.
 
+NginxTestCase
+------------------
+
+Configuration
+''''''''''''''''''
+
+By default, the ``NginxTestCase`` starts and stops NGINX server for you and delete
+all cached contents. You have to set your NGINX configuration file. 
+
+These are all the configurations you can change
+
+======================= ========================= ============ ==========================================
+Constant                Getter                    Default      Description
+======================= ========================= ============ ==========================================
+``NGINX_FILE``          ``getConfigFile()``                    your NGINX configuration file
+``NGINX_BINARY``        ``getBinary()``           ``nginx``    your NGINX binary
+``NGINX_PORT``          ``getCachingProxyPort()`` ``8088``     port NGINX listens to
+``NGINX_CACHE_PATH``    ``getCacheDir()``                      NGINX cache dir path
```

Comment:
is there a default for this? is this specified in the configuration file? (we should make it clear why you have to specify this, and if this must match what is configured in the config file)